jerry-core/debugger/jerry-sha1.c

```diff
@@ -126,10 +125,10 @@ jerry_sha1_process (jerry_sha1_context *sha1_context_p, /**< SHA-1 context */
 )
 
 #define P(a, b, c, d, e, x) \
-{ \
+( \
```

Comment:
I don't think you can use `(` here because of the `;`. You can use do-while(0) if you wish.